recipes/config/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "config" %}
```

Comment:
This name is not confusing at all :smile:

Reply:
Agreed :laughing: